testFramework/com/twitter/intellij/pants/testFramework/PantsIntegrationTestCase.java

```diff
@@ -493,6 +493,17 @@ protected ScalaTestRunConfiguration generateScalaRunConfiguration(String moduleN
     return runConfiguration;
   }
 
+  protected void gitCleanExampleDir() throws ExecutionException {
```

Comment:
It looks like this also does a git reset --hard and a rm of dist. I think it could have a better name.